projects/2019_12_14_pkt-whitepaper-website.md

```diff
@@ -372,3 +372,589 @@ The applicant understands that the Network Steward is not a legal entity and no
 [198](https://pkt-insight.cjdns.fr/#/PKT/pkt/tx/6912729413f16888cacacb006de0c3b3b70a3ad703bedfdc0bf5a478eaee2c57)
 [199](https://pkt-insight.cjdns.fr/#/PKT/pkt/tx/fcd0c40dccd650a271c9311b5ce302019455f8c02445faae1dee9e90b811a13f)
 [200](https://pkt-insight.cjdns.fr/#/PKT/pkt/tx/ec9ca663ea47b50aebaaa5b2d064959111ab09635cb26ecbad6bcb884cd34a8b)
+
+#PKT website for review and notes. A request for feedback from the Network Stewards
+
+------------------------ BEGINNING OF PROJECT ------------------------
+
+{PAGE}
+#Homepage
+Menu
+Download whitepaper button on right-side of menu
+Featured: animated gif of nodes connecting around the world (a sphere earth) in a web-like fashion with packets moving between them. 
+Text headers says: “The world’s first network-based blockchain”
+Network Stats info (pull data from blockexplorer)
+Min bandwidth req. 1mb/second
+Transactions per day
+Block number
+Minimum Bandwidth requirements
+Transactions 
+
+##Bandwidth-Hard
+Allocating network bandwidth is required to earn mining rewards.
+ 
+##Networking
+The more connected nodes the faster the network becomes.
+
+##Scalability
+Meets the demands of network operators and infrastructure providers.  
+
```

Comment:
These three points are  super important. It would  be great to link these to more information about these topics.

Also, I'd like these contextualized so that the first time reader understands why they are important.

Something like:

PKT is a new type of networking protocol that enables hyper secure data transfer end-to-end and point-to-point. It becomes more robust and more reliable the the more people use it, and it allows people to buy and sell bandwidth using micropayments.

This is possible because it is:
- Encrypted by default
  short explanation

- Distributed
  short explatation

- Bandwidth hard
  short explatanio